README.md

```diff
@@ -1,6 +1,6 @@
-[![](https://vsmarketplacebadge.apphb.com/version/mlxprs.mlxprs.svg)](https://marketplace.visualstudio.com/items?itemName=mlxprs.mlxprs)
+<!-- [![](https://vsmarketplacebadge.apphb.com/version/mlxprs.mlxprs.svg)](https://marketplace.visualstudio.com/items?itemName=mlxprs.mlxprs)
```

Comment:
Note: The VSCE packager rejected these SVG images. I will write a story to fix this.